### DISCUSSION TITLE
Print Replica

### DISCUSSION THREAD

**Discussion by tomsem** [apprenticeharper/DeDRM_tools#1665] [Q&A]
Can someone confirm the latest versions of DeDRM tools (I’m on release just prior to current one) do not support decrypting Kindle Print Replica format (e.g. using Kindle for PC or Mac)? Is this likely to be supported in a future update? 

I take it this was possible with prior versions of calibre/DeDRM and Kindle for PC/Mac. However, assuming that’s true, it isn’t particularly something I would bother with (I think I have only 2 of these).

I can download .azw4 file from my Content page (for Fire Kindle app) but take it that cannot be decrypted either.

I’m a little confused about Kindle for Android: the calibre plugin has a way to add keys for it, as if it were possible to decrypt the content, however on the Wiki here it says ‘nope’. Note that more recent Fires seem to have a Kindle app much the same as Kindle for Android (with same version). But my only interest here would be to get a Print Replica file DeDRMed.

As it is I am downloading from Content page (Transfer via USB) and that’s working fine for everything but PR.

**Comment by ElleKayEm** [comment]
I can't confirm myself but I believe I have seen people having issues with calibre 5/DeDRM 7 and AZW4.  Should work with calibre 4.23/DeDRM 6.8.1 as long as it's a purchased book and not a rental.  To decrypt any book for download & transfer via USB, you would need to choose an E-ink Kindle (not a Fire).  And again, an AZW4 will probably need calibre 4.23 and DeDRM 6.8.1.  The method for the Android app no longer works.  You needed a backup file from the app which Amazon no longer allows.

**Reply by tomsem** [reply]
I did try calibre 5/DeDRM 7 but it didn’t work with current version of Kindle for Mac. It isn’t clear to me that the older version of calibre and tools would work with the current version, either. I would need to set stuff up on my older Mac, I might give it a try but it is a low priority.

Print Replica format is not available for Kindles, of course. I mentioned Fire only because those are the only devices you can download AZW4 for transfer via USB from the Content page. Seems like it could be relatively easy to figure out how to decrypt downloaded Fire content in general by diffing this with Kindle download (I tried adding Fire SN but that didn’t work).

**Reply by ElleKayEm** [reply]
Work on DeDRM 7 (Python 3) is still ongoing for some of the less common file types like AZW4.  The older Python 2 versions of calibre and DeDRM are more likely to work on those types.  Non KFX books can be decrypted from the latest Kindle for PC/Mac.  Extracting a key from Fires is the issue.  With E-ink Kindles, it is based on the serial number, but not with the Fires.
